cumulus/parachains/runtimes/testing/penpal/src/lib.rs

```diff
@@ -841,6 +909,8 @@ construct_runtime!(
 		PoolAssets: pallet_assets::<Instance3> = 52,
 		AssetConversion: pallet_asset_conversion = 53,
 
+		Revive: pallet_revive = 60,
```

Comment:
`Contracts: pallet_revive = 60,`
`SmartContracts: pallet_revive = 60,`
`PolkaVmContracts: pallet_revive = 60,`
???

or is `Revive: pallet_revive` final decision also for AssetHubKusama/Polkadot?

Reply:
I don't know. For `AssetHubWestend` it's `Revive: pallet_revive`. I copied it from there. I suppose that's the go-to name for now.